internal/controller/shipwrightbuild_controller.go

```diff
@@ -59,6 +60,20 @@ func (r *ShipwrightBuildReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		return err
 	}
 
+	// Remove runAsUser and runAsGroup from a Deployment container's security context
+	// Insert Openshift Service CA annotations in service and CRD
+	if r.Manifest, err = r.Manifest.Transform(
+		common.RemoveRunAsUserRunAsGroup,
+		common.InjectAnnotations("Service", common.ShipwrightWebhookServiceName, map[string]string{
+			openshiftserviceca.ServingCertSecretAnnotation: common.ShipwrightWebhookCertSecretName,
+		}),
+		common.InjectAnnotations("CustomResourceDefinition", "", map[string]string{
+			openshiftserviceca.InjectCABundleAnnotationName: "true",
+		}),
+	); err != nil {
+		return err
+	}
+
```

Comment:
I think this is the right move.

I initially thought we had logic in shipwright-io/operator to deal with this OpenShift-ism, but it looks like it isn't 100% there in the [upstream code](https://github.com/shipwright-io/operator/blob/release-v0.13/controllers/shipwrightbuild_controller.go#L194-L201). There might have been undocumented changes to `release.yaml` to add these annotations by hand in Builds 1.0.z. :face_with_diagonal_mouth: 

We otherwise have the right environment variable (`PLATFORM`) set to invoke this OpenShift-specific logic in the Shipwright code.